### ISSUE TITLE
consider two-column default template

### ISSUE THREAD

**Issue by jrhizor** [stoat-dev/stoat-action#113]
It may be beneficial to make the overall comment shorter by using two columns:

---

**Easy and customizable dashboards for your build system**. Learn more about [Stoat ↗︎](https://stoat.dev)

| Static Hosting | Job Runtime | 
| :--------------|:-------------|
|<table> <thead> <tr> <th style="text-align:left">Name</th> <th style="text-align:center">Commit</th> <th style="text-align:center">Status</th> </tr> </thead> <tbody> <tr> <td style="text-align:left"><a href="https://stoat-dev--stoat-action--4862dc6--docs.stoat.page">Documentation</a></td> <td style="text-align:center">4862dc6c5e23989f2784297f3175c5f846be04a0</td> <td style="text-align:center">✅</td> </tr> <tr> <td style="text-align:left"><a href="https://stoat-dev--stoat-action--50040c1--test-cover-d630.stoat.page">CLI Test Coverage</a></td> <td style="text-align:center">50040c1c9e1023b03139a524209e3d06e3815290</td> <td style="text-align:center">✅</td> </tr> <tr> <td style="text-align:left"><a href="https://stoat-dev--stoat-action--f2b1008--test-cover-65ca.stoat.page">Action Test Coverage</a></td> <td style="text-align:center">f2b10080d2f746a15577e10a78d83877d209d4b2</td> <td style="text-align:center">✅</td> </tr> </tbody> </table> |<a href="https://quickchart.io/chart/render/sp-8fc90249-417e-4da5-a195-c2cddd5a2a7b" target="_blank"><img src="https://quickchart.io/chart/render/sp-8fc90249-417e-4da5-a195-c2cddd5a2a7b" height="300" /></a>|

[![debug](https://shields.io/badge/-debug-lightgrey)](https://www.stoat.dev/debug/github/1371837091?ghDefaultBranch=main)

---

I'd be curious what everyone thinks about this vs the current default.

**Comment by tuliren** [comment]
The table in table on the left is a bit weird.

**Comment by jrhizor** [comment]
Yeah. Maybe if we share the same top bar? Not sure how that will look with GitHub formatting or as we introduce other plugins.
